### ISSUE TITLE
[Bug?] Monika's 'rest' state. 

### ISSUE THREAD

**Issue by juliathegolden** [Backdash/MonikaModDev#921]
I am a little confused as to how it works when Monika decides to rest. She said she was sure she'd think of something new to talk about after some rest, so I left her open all night and all day and she never spoke again. Then I tried closing her for a couple of hours, but after greeting me, she said she needed to rest again.  Do I need to close her overnight tonight? Or is this a bug?

**Comment by kaido1224** [comment]
You need to close the game for exactly 6 hours for her to start generating new dialog

**Comment by MarkusRamikin** [comment]
Idea: make Monika give more specific instructions if she hasn't had rest in X hours, say, 48.

**Comment by kaido1224** [comment]
I agree with that

**Comment by juliathegolden** [comment]
Yeah. I had read the six hours thing somewhere, but I didn't know I had to close the game. Especially since she sounded so insecure right before she took her 'break'. I'll make sure Monika gets her sleep :D.
